io-engine/src/core/logical_volume.rs

```diff
@@ -31,6 +32,9 @@ pub trait LogicalVolume {
     /// Return the committed size of the Logical Volume in bytes.
     fn committed(&self) -> u64;
 
+    /// Get BlobPtr from spdk_lvol.
```

Comment:
```suggestion
    /// Get BlobPtr.
```

This is fine for now but I think in the future we'll need to refactor this as we're bringing already the second spdk specific method to an interface which is meant to be generic.